packit_service/constants.py

```diff
@@ -46,6 +46,15 @@
     "approved_manually": "approved_manually",
 }
 
+INSTALLATION_INSTRUCTIONS = (
+    "You can install the built RPMs by following these steps:\n\n"
+    "* `sudo yum install -y dnf-plugins-core` on RHEL 8\n"
+    "* `sudo dnf install -y dnf-plugins-core` on Fedora\n"
+    "* `dnf copr enable {owner}/{project_name}`\n"
+    "* And now you can install the packages.\n"
+    "\nPlease note that the RPMs should be used only in a testing environment."
```

Comment:
Is this line valid also for the branch/release builds? Is it possible to add this not only for PRs?

Reply:
You mean the line `dnf copr enable {owner}/{project_name}`? Since the `owner` and `project_name` are for the copr project, we have it in db for branches and releases as well. Or am I missing something?

Reply:
No, I mean the line 55 -- `"\nPlease note that the RPMs should be used only in a testing environment."`.

That it does not need to be testing version.

Reply:
Oh, I see. I will change that